docs/src/docs/asciidoc/gradle-plugin.adoc

```diff
@@ -324,6 +324,30 @@ Same can be achieved by setting corresponding DSL option, althought this isn't r
 
 The generated configuration files will be found in the `${buildDir}/native/agent-output/${taskName}` directory, for example, `build/native/agent-output/run`.
 
+=== Disabling test support
+
+There are cases where you might want to disable test support:
+
+- you don't actually want to run your tests in native mode
+- your library or application uses another test framework than JUnit Platform (currently this plugin only supports testing with JUnit Platform)
```

Comment:
clarify JUnit 5 since we have tickets as well asking for JUnit 3 support

Reply:
done